pimd/pim_iface.c

```diff
@@ -1481,6 +1481,12 @@ void pim_if_create_pimreg(struct pim_instance *pim)
 {
 	char pimreg_name[IFNAMSIZ];
 
+	if (PIM_DEBUG_TRACE)
```

Comment:
This commit a bit looks strange sneaked into docker-related PR ;-)

Reply:
I actually used this to debug the problems in the Pim test. in any case check out PR #16124, I've moved all the non-docker changes there. :)